### ISSUE TITLE
Maximum methods per class

### ISSUE THREAD

**Issue by sbradl** [MO2k4/CleanCode#22]
I think this should only take public and maybe protected methods into account. At the moment it counts private methods too.

Nice work btw .)

**Comment by MO2k4** [comment]
I think that is a tricky question, because would you consider a class with 2-3 public methods and 10-100 private methods clean? I don't think that a class should be divided by access identifier, because the overall size of the class increases and so would the responsibility.

**Comment by sbradl** [comment]
Yeah indeed it is tricky. Without any actual numbers I would rather tolerate 20 private methods but not 20 public ones. But 1 public method with 20 private ones seems off too.
